### ISSUE TITLE
1.6.4 crashes on startup

### ISSUE THREAD

**Issue by ghengeveld** [mingo-app/mingo#312]
I've been using 1.5.0 succesfully so far, but's it's been nagging for an upgrade, which I finally installed today. However, 1.6.4 crashes on startup. I've tried downgrading back to 1.5.0 (I still had the dmg in my downloads folder), but that didn't help. I removed `~/Library/Application Support/mingo` and tried again, but that didn't resolve the problem either. I also tried reinstalling Rosetta.

How can I get Mingo to work again?

I'm running Mac OS Monterrey on Macbook Pro M1 via Rosetta 2.

**Comment by ghengeveld** [comment]
<img width="275" alt="Screenshot 2021-12-06 at 20 14 20" src="https://user-images.githubusercontent.com/321738/144907737-7272e2df-a6b9-4709-9bd4-4e618a37320c.png">

```
[2021-11-15 23:14:29.041] [info]  App starting...
[2021-11-19 09:23:18.142] [info]  App starting...
[2021-11-22 10:46:39.815] [info]  App starting...
[2021-11-25 10:02:18.624] [info]  App starting...
[2021-11-25 10:02:47.737] [info]  meta
[2021-11-25 10:03:49.106] [info]  App starting...
[2021-11-25 10:04:04.359] [info]  meta
[2021-11-25 13:00:35.607] [info]  App starting...
[2021-11-25 21:53:21.177] [info]  App starting...
[2021-11-29 12:17:09.066] [info]  App starting...
[2021-11-29 19:54:40.276] [info]  App starting...
[2021-12-02 17:32:58.511] [info]  App starting...
[2021-12-06 13:53:33.772] [info]  App starting...
[2021-12-06 13:53:49.111] [info]  meta
[2021-12-06 13:54:00.861] [info]  App starting...
```

Not much interesting there. Here's an excerpt from the crash report with what might be useful:

```
Exception Type:  EXC_BREAKPOINT (SIGTRAP)
Exception Codes: 0x0000000000000001, 0x00007ff7ffe4caf8
Exception Note:  EXC_CORPSE_NOTIFY
Termination Reason: SIGNAL 5 Trace/BPT trap: 5
Terminating Process: exc handler [88420]

Triggered by Thread:  0

Application Specific Information:
rosetta error: /var/db/oah/279281325309952_279281325309952/42d389f280805e79dd6db26933047b3c5ec154b65fb5cb3551f2170678687b92/Mingo.aot: attachment of code signature supplement failed: 1
```

**Comment by tothradoslav** [comment]
Seems like a specific issue with M1/Rosetta, even though you are saying you used it with v1.5. 

After googling the issue, I found 2 points that should help, but I cannot reproduce your issue as no one in our team had it. Please, if you could try one of these (if applicable) and let me know:

1. restart OS - this sounds funny, but it's probably an uptime issue with the new macos
2. upgrade the OS - not sure if you are using the latest version of Monterey

Please let me know.

**Comment by ghengeveld** [comment]
Thanks, restarting Mac OS resolved the issue (using 1.6.4). I guess I should've backed up my settings though. Oh well...
I was already on the latest Monterey.

**Comment by tothradoslav** [comment]
I'm glad and actually surprised it worked :)

BTW, you can save your entire Mingo state from the "File" menu and import (use) it whenever needed or on a new machine.
